pyproject.toml

```diff
@@ -10,7 +10,7 @@ packages = [
 include = ["config/*.json"]
 
 [tool.poetry.dependencies]
-python = '>= 3.8, < 4.0'
+python = '>= 3.10, < 4.0'
```

Comment:
I think that it's a tradeoff. 
**Pro for 4.0:** Less restrictive for contributors. It might be annoying if they have to install a very specific python version. It also makes it a bit easier to do the backend python update because it can be done independently of contrib (like we've done here).
**Pro for 3.11:** Easier to guarantee that it will work on the backend as well. 

I'm fine with both. What do you think?

Reply:
Since we already tell them they need Python 3.10 and rely on that I am in favour of the more restrictive versioning, mostly so we don't end up using some fancy feature in Python 3.15 or something in a parser that won't be supported at all by the backend.

The backend update is a good point but I think if we just did them the other way around and updated contrib first (with a new range) that would solve that issue right?

But honestly I don't think it is such a big deal, especially not if we get the parsers updated with matching tests.